q2_alignment/_mafft.py

```diff
@@ -26,7 +26,8 @@ def run_command(cmd, output_fp, verbose=True):
         subprocess.run(cmd, stdout=output_f, check=True)
 
 
-def _mafft(sequences_fp, alignment_fp, n_threads, parttree, addfragments):
+def _mafft(sequences_fp, alignment_fp, n_threads, parttree, addfragments,
+           keeplength):
```

Comment:
```suggestion
           keeplength=False):
```

Reply:
I left this as is for now as it is similar to `parttree`. I just I just followed that lead of adding `False` in the `mafft` function. I can change that it that is the better way to go. :-)